homeassistant/components/tradfri/base_class.py

```diff
@@ -23,21 +23,19 @@
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity import DeviceInfo, Entity
 
-from .const import DOMAIN, SIGNAL_GW
+from .const import DOMAIN, SIGNAL_GW, TIMEOUT_API
 
 _LOGGER = logging.getLogger(__name__)
 
 
-def handle_error(
-    func: Callable[[Command | list[Command]], Any]
-) -> Callable[[str], Any]:
+def handle_error(func: Callable[..., Any]) -> Callable[[str], Any]:
```

Comment:
The typing of the return value is incorrect. Either we need to type it explicitly and correctly or we need to use generics and except arbitrary parameters. See the mypy cheat sheet:

https://mypy.readthedocs.io/en/stable/generics.html#declaring-decorators

Reply:
Done.